packages/ui/src/views/layouts/SandboxLayout.tsx

```diff
@@ -79,56 +78,61 @@ function Main({
   hasHeader,
   hasSubHeader,
   hasLeftPanel,
-  hasLeftSubPanel
+  hasLeftSubPanel,
+  leftSubPanelWidth = 300
 }: {
-  children: React.ReactNode
+  children: ReactNode
   fullWidth?: boolean
   className?: string
   hasHeader?: boolean
   hasSubHeader?: boolean
   hasLeftPanel?: boolean
   hasLeftSubPanel?: boolean
+  leftSubPanelWidth?: number
 }) {
   const paddingTopClass =
     hasHeader && hasSubHeader ? `pt-[calc(55px+45px)]` : hasHeader ? 'pt-[55px]' : hasSubHeader ? 'pt-[45px]' : ''
 
-  const paddingLeftClass =
-    hasLeftPanel && hasLeftSubPanel
-      ? 'pl-[calc(220px+300px)]'
-      : hasLeftPanel
-        ? 'pl-[220px]'
-        : hasLeftSubPanel
-          ? 'pl-[300px]'
-          : ''
+  const paddingLeftStyle: CSSProperties = {
+    paddingLeft:
+      hasLeftPanel && hasLeftSubPanel
+        ? `calc(220px + ${leftSubPanelWidth}px)`
+        : hasLeftPanel
+          ? '220px'
+          : hasLeftSubPanel
+            ? `${leftSubPanelWidth}px`
+            : ''
+  }
```

Comment:
Just curious; is the move to `style` instead of a Tailwind class because the relevant utility class would be stripped from the output since it's determined at runtime and the Tailwind utility wouldn't find it in the source?

Reply:
The issue is that Tailwind cannot generate custom classes during the build process because the value depends on props. That's why I'm using styles instead.

Reply:
Yeah, that makes sense 👍🏻